kolibri/core/content/serializers.py

```diff
@@ -122,6 +124,52 @@ class Meta:
         )
 
 
+class PublicChannelSerializer(KolibriModelSerializer):
```

Comment:
these serializer fields are mapped from the conversion response
https://github.com/learningequality/kolibri/blob/develop/kolibri/core/content/api.py#L655